internal/brokers/broker_test.go

```diff
@@ -95,9 +99,10 @@ func TestGetAuthenticationModes(t *testing.T) {
 
 		wantErr bool
 	}{
-		"Get authentication modes and generate validators":          {sessionID: "success", supportedUILayouts: []string{"required-value", "optional-value"}},
-		"Get authentication modes and ignores invalid UI layout":    {sessionID: "success", supportedUILayouts: []string{"required-value", "missing-type"}},
-		"Get multiple authentication modes and generate validators": {sessionID: "GAM_multiple_modes", supportedUILayouts: []string{"required-value", "optional-value"}},
+		"Get authentication modes and generate validators":                     {sessionID: "success", supportedUILayouts: []string{"required-value", "optional-value"}},
+		"Get authentication modes and generate validator ignoring whitespaces": {sessionID: "success", supportedUILayouts: []string{"layout-with-spaces"}},
```

Comment:
That was almost finale :p I would say ignoring whitespaces in accepted values or so. (we don’t allow whitespace around required/optional)